.local-manifest/sablier/sablier-deployment.yaml

```
@@ -29,3 +29,6 @@ spec:
             - name: project
               subPath: .local-dev/config/sablier.yaml
               mountPath: /etc/sablier/sablier.yaml
+            - name: themes
+              subPath: dashboard/sablier-theme
+              mountPath: /etc/sablier/themes
```

`themes`というvolumeが定義されてないので、こっちも`project`にしてしまうのがよさそうです
```suggestion
            - name: project
              subPath: dashboard/sablier-theme
              mountPath: /etc/sablier/themes
```